packages/docs/src/styles/pages.css

```
@@ -372,6 +372,155 @@
+  padding: 1rem;
+  background: var(--bg-secondary);
+  border-radius: 8px;
+  border: 1px solid var(--border-color);
+  text-decoration: none;
+  color: var(--text-primary);
+  transition: transform 0.2s, box-shadow 0.2s;
+}
+
+.getting-started-card:hover {
+  transform: translateY(-2px);
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.08);
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)